beater/api/mux.go

```diff
@@ -166,6 +180,13 @@ func (r *routeBuilder) backendIntakeHandler() (request.Handler, error) {
 	return middleware.Wrap(h, backendMiddleware(r.cfg, r.authenticator, r.ratelimitStore, intake.MonitoringMap)...)
 }
 
+func (r *routeBuilder) backendOtlpIntakeHandler(handler http.HandlerFunc, monitoringMap map[request.ResultID]*monitoring.Int) func() (request.Handler, error) {
+	return func() (request.Handler, error) {
+		h := intake.OtlpHandler(handler)
+		return middleware.Wrap(h, backendMiddleware(r.cfg, r.authenticator, r.ratelimitStore, monitoringMap)...)
+	}
+}
```

Comment:
```suggestion
func (r *routeBuilder) otlpHandler(handler http.HandlerFunc, monitoringMap map[request.ResultID]*monitoring.Int) func() (request.Handler, error) {
	return func() (request.Handler, error) {
		h := func(c *request.Context) {
			handler.ServeHTTP(c.ResponseWriter, c.Request)
		}
		return middleware.Wrap(h, backendMiddleware(r.cfg, r.authenticator, r.ratelimitStore, monitoringMap)...)
	}
}
```

I don't think "backend" or "intake" are relevant here: this endpoint should be usable for any OTLP client (backend, RUM, etc.), and as mentioned in another comment "intake" is a reference to the Elastic APM agent intake protocol.

Reply:
Inplemented in 51cdb119d